src/filesystem/impls/appshell/node/node_modules/fsevents_win/fsevents_win.cpp

```diff
@@ -337,7 +339,7 @@ DWORD WINAPI NodeFSEvents::Run(LPVOID lpData)
 	{
 		// iterate watching for directory changes until we're signalled to stop
 		BOOL bContinue = TRUE;
```

Comment:
This variable should be taken off entirely. Since you removed the code at lines 386-390, `bContinue` never changes the value and having it around is excessive and not needed.